src/expat/configure.ac

```diff
@@ -45,18 +45,14 @@ dnl
 dnl If the API changes incompatibly set LIBAGE back to 0
 dnl
 
-LIBCURRENT=7
-LIBREVISION=0
-LIBAGE=6
+LIBCURRENT=7   # sync
+LIBREVISION=2  # with
+LIBAGE=6       # CMakeLists.txt!
 
 AC_CONFIG_HEADER(expat_config.h)
 
 sinclude(conftools/ac_c_bigendian_cross.m4)
 
-
-dnl Check for many C compilers (have to do this before libtool stuff)
-AC_BAKEFILE_PROG_CC
```

Comment:
Nope. It is part of changes in expat.
The first commit has the original expat code, and my only change was to redefine XML_STATIC in the same header as 2.1.1.